[IIGO] [2/3] More Analysis and Evaluation (#65)
SOMAS2020/SOMAS-Final-Report: 05_iigo/index.tex

    The enforcement of collective choice rules is discussed in with Ostrom's 4th principle.
    
    \item Monitoring, of both state conditions and appropriator behaviour, is by appointed agencies, who are either accountable to the resource appropriators or are appropriators themselves.
    
    The first set of monitoring is performed by the Judge. It is described in Action Arena 3. While the Judge is always a resource appropriator, under an institutional state of tyranny or oligarchy, where the role of the Judge is only ever in the hands of one or a few islands, the decision group consists of only one island. Moreover, in the current IIGO implementation, there is no way to monitor the behaviour of the Judge in this regard, there is no way to confirm that the monitoring results announced by the Judge are concurrent with the state and events of the game. This is an issue that should be addressed in future iterations of IIGO.

>     The second set of monitoring is that which roles perform in what is referred to as the Accountability Cycle (see Section~\ref{sec:accountability}). Whether roles abide by the collective choice rules in play is monitored by the roles themselves. The most notable flaw in this section is the one that was alluded to in the repetitive nature of elections and monitoring. There is a possible outcome of the institution where the agent responsible for the appointment, disregards the election process and appoints themselves. The reverse nature of the transfer-of-power and monitoring cycles means that the same agent is then responsible for monitoring their own inappropriate behaviour. Within one degree of monitoring, where monitoring itself is not monitored, the agent would be accountable to only itself. 

Reviewer: ```suggestion
    The second set of monitoring is that which roles perform in what is referred to as the Accountability Cycle (see Section~\ref{sec:accountability}). Whether roles abide by the collective choice rules in play is monitored by the roles themselves. The most notable flaw in this section is the one that was alluded to in the repetitive nature of elections and monitoring. There is a possible outcome of the institution where the agent responsible for the appointment disregards the election process and appoints themselves. The reverse nature of the transfer-of-power and monitoring cycles means that the same agent is then responsible for monitoring their own inappropriate behaviour. Within one degree of monitoring, where monitoring itself is not monitored, the agent would be accountable to only itself. 
```

    \item A flexible scale of graduated sanctions for resource appropriators who violate communal rules; i.e. violation should be proportional to the severity, frequency or necessity of the offence.
    
    While the agents are given a flexible system of graduated sanctions to utilise when in the role of the Judge in the form of sanction tiers and setting the severity of offence, these decisions are entirely and freely made by the Judge. There are no institutional "guardrails", there is no Zone of Dignity for this process. An agent could choose to not sanction itself or to not sanction severe and frequent offences while still heavily sanctioning agents which are minor one-time offenders. While the sanctioning mechanism is present, the lack of a standard protocol indicates that the current implementation of IIGO is lacking with regard to Ostrom's fifth principle.

>     When looking at the sanctions relating to offences that roles can make when in power, IIGO is also lacking in this regard. The worst punishment any agent can receive for not abiding by the rules that govern the decision-making in the 3 government branches is that the agent can be let go from the role prematurely. Hence the only thing an agent can lose is the potential for future salary. Hence there are situations where agents could calculate that the optimal strategy is to always abuse these powers for their own benefit until caught.

Reviewer: ```suggestion
    When looking at the sanctions relating to offences that roles can make when in power, IIGO is also lacking in this regard. The worst punishment any agent can receive for not abiding by the rules that govern the decision-making in the 3 government branches is that the agent can be let go from the role prematurely. Hence the only thing an agent can lose is the potential for future salary. Hence, there are situations where agents could calculate that the optimal strategy is to always abuse these powers for their own benefit until caught.
```

>     When looking at the sanctions relating to offences that roles can make when in power, IIGO is also lacking in this regard. The worst punishment any agent can receive for not abiding by the rules that govern the decision-making in the 3 government branches is that the agent can be let go from the role prematurely. Hence the only thing an agent can lose is the potential for future salary. Hence, there are situations where agents could calculate that the optimal strategy is to always abuse these powers for their own benefit until caught.

Reviewer: ```suggestion
    When looked at the sanctions related to the offences that roles can make when in power, IIGO is also lacking in this regard. The worst punishment any agent can receive for not abiding by the rules that govern the decision-making in the 3 government branches is that the agent can be let go from the role prematurely. Hence the only thing an agent can lose is the potential for future salary. Hence, there are situations where agents could calculate that the optimal strategy is to always abuse these powers for their own benefit until caught.
```

>     When looked at the sanctions related to the offences that roles can make when in power, IIGO is also lacking in this regard. The worst punishment any agent can receive for not abiding by the rules that govern the decision-making in the 3 government branches is that the agent can be let go from the role prematurely. Hence the only thing an agent can lose is the potential for future salary. Hence, there are situations where agents could calculate that the optimal strategy is to always abuse these powers for their own benefit until caught.

Reviewer: ```suggestion
    When looked at the sanctions related to the offences that roles can make when in power, IIGO is also lacking in this regard. The worst punishment any agent can receive for not abiding by the rules that govern the decision-making in the 3 government branches is that the agent can be let go from the role prematurely. Hence the only thing an agent can lose is the potential for future salary. Hence, there are situations where agents could calculate that the optimal strategy is to always abuse these powers for their own benefit until they are caught.
```

    \item Universal access to ‘fair’, rapid, low-cost, light-weight conflict-resolution mechanisms.
    

>     When looking at the current implementation of IIGO we can notice that monitoring and enforcement are heavily linked and both decisions for operational-rule offences are held by the Judge. A sanctioned island would have no way of appealing the decision made by the Judge. Hence there is no conflict-resolution mechanism. Moreover, there is no "innocent until proven guilty" mechanism. One way to remedy this, without introducing a Zone of Dignity for the Judge's decisions, which can then be monitored. An alternative could be to introduce an Alternative Dispute Resolution protocol, with respective meta-rules, as well as the idea of a court in cases where the Alternative Dispute Resolution would necessitate litigation. 

Reviewer: ```suggestion
    When looking at the current implementation of IIGO we can notice that monitoring and enforcement are heavily linked and both decisions for operational-rule offences are held by the Judge. A sanctioned island would have no way of appealing the decision made by the Judge. Hence, there is no conflict-resolution mechanism. Moreover, there is no "innocent until proven guilty" mechanism. One way to remedy this, without introducing a Zone of Dignity for the Judge's decisions, which can then be monitored. An alternative could be to introduce an Alternative Dispute Resolution protocol, with respective meta-rules, as well as the idea of a court in cases where the Alternative Dispute Resolution would necessitate litigation. 
```
